strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenArtifactControllerTest.java

```diff
@@ -211,80 +236,88 @@ private static void createPluginXmlFile(String groupId,
         transformer.transform(source, result);
     }
 
-    public static Set<MutableRepository> getRepositoriesToClean()
-    {
-        Set<MutableRepository> repositories = new LinkedHashSet<>();
-        repositories.add(createRepositoryMock(STORAGE0, REPOSITORY_RELEASES1, Maven2LayoutProvider.ALIAS));
-        repositories.add(createRepositoryMock(STORAGE0, REPOSITORY_RELEASES2, Maven2LayoutProvider.ALIAS));
-        repositories.add(createRepositoryMock(STORAGE0, REPOSITORY_SNAPSHOTS, Maven2LayoutProvider.ALIAS));
-
-        return repositories;
-    }
-
     @Override
     @BeforeEach
     public void init()
             throws Exception
     {
         super.init();
 
-        GENERATOR_BASEDIR = new File(ConfigurationResourceResolver.getVaultDirectory() + "/local");
+        MockitoAnnotations.initMocks(this);
+        defaultMavenArtifactDeployer = buildArtifactDeployer(Paths.get(""));
 
         MutableMavenRepositoryConfiguration mavenRepositoryConfiguration = new MutableMavenRepositoryConfiguration();
-        mavenRepositoryConfiguration.setIndexingEnabled(true);
+        mavenRepositoryConfiguration.setIndexingEnabled(false);
```

Comment:
I think the default for this is `false`, so it should even be safe to remove this line.